src/commandline_content.ts

```diff
@@ -19,14 +19,15 @@ const cmdline_logger = new Logger("cmdline")
 
 let cmdline_iframe: HTMLIFrameElement = undefined
 let enabled = false
+let initted = false
```

Comment:
Isn't `initted` useless since its state will always be the same as `!!cmdline_iframe` and we're already checking that `cmdline_iframe === undefined` everywhere we're checking `!initted`?